web/src/lib/components/ItemView.svelte

```diff
@@ -335,18 +336,20 @@
                                     <p class="text-4xl text-center pt-24">
                                         {#if item instanceof Auction}
                                             Your auction is live! <br />
-
-                                            <div class="text-xl text-center my-2 mt-10 mb-10">
-                                                Now let your audience know!
-                                                &nbsp;
-                                                <TweetButton tweetURL={tweetURL} />
-                                            </div>
                                         {:else if item instanceof Listing}
                                             Your listing is live! <br />
                                             <br />
                                             Note: You can still edit it, by going to <a class="link" href="/stall/{$user.nym}">My stall</a>!
                                         {/if}
                                     </p>
+                                    <div class="text-xl text-center my-2 mt-10 mb-10">
+                                        Now let your audience know!
+                                        &nbsp;
+                                        <TweetButton tweetURL={tweetURL} />
+                                        {#if ! isProduction() }
```

Comment:
Actually I don't really like passing the URL of the window here... that makes the <NostrButton not very reusable.

And while looking now at how tweetURL is built ... I found a bug there. we always use /auctions - so what happens when somebody wants to tweet out a listing? It will use the wrong link...

Which makes me think - maybe we should just have a getUrl() function on IEntity, which Auction and Listing will implement, and we would use that both for Twitter and Nostr. What do you think?